packages/shared/src/modules/users/libs/enums/user-validation-rule.enum.ts

```diff
@@ -3,7 +3,7 @@ const UserValidationRule = {
 	EMAIL_MINIMUM_LENGTH: 6,
 	NAME_MAXIMUM_LENGTH: 50,
 	NAME_MINIMUM_LENGTH: 3,
-	NAME_PATTERN: /^[\sA-Za-z]*$/,
+	NAME_PATTERN: /^[\sA-Za-z-]*$/,
```

Comment:
should we also ensure that hyphen is surrounded by characters?
`NAME_PATTERN: /^(?!.*--)[A-Za-z]+(-[A-Za-z]+)*$/,`

Reply:
Okay, I see. We don’t want to complicate it. Got it :+1: